### DISCUSSION TITLE
Getting image creation time when no EXIF data

### DISCUSSION THREAD

**Discussion by AlquistArj** [ImageMagick/ImageMagick#3082] [General]
I'm using Magick from c#. I have some .jpg's that are being downloaded from the web. Some of the .jpg files have EXIF data and I can easily read the DateTime when the image was created from the various different tags. However, some .jpg's don't have EXIF data yet Magick is still able to successfully set the File Creation time when it writes the file to disk (using MagickImage.Write). Where does it get the file creation date from? Can I also access that creation time?

**Comment by urban-warrior** [comment]
ImageMagick pulls the creation time from the image file you specify.  If you download and image off the Internet, it typically preserves the creation date of the file.  The file creation is far from an authoritative creation date of an image given the image file might have been copied a number of times and the creation date changed.  Generally, only a creation date tag in EXIF or some other profiles is consider authoritative.
